### DISCUSSION TITLE
Hex Color Codes

### DISCUSSION THREAD

**Discussion by astr0phile777** [stoatchat/discussions#668] [Feature Suggestions]
Hi! I tried to look up related suggestions and saw none, apologies I am new to github!
I wanted to suggest adding the ability to set colors using a hex code instead of scroll wheels as they are pretty inaccessible!
For both profile accent colors and also display name colors in servers and such there is no option to place a hex code so I have to eyeball to get the colors I want.
Thankfully you can use hexs for themes if you go to "theme overrides" but I think the option should be there without having to search for it! Thank you!

**Comment by Rexogamer** [comment]
What browser are you using?
